src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -68,6 +68,11 @@ public function load(array $configs, ContainerBuilder $container)
 
         if ($container->hasParameter('kernel.debug') && $container->getParameter('kernel.debug')) {
             $loader->load('security_debug.xml');
+
+            if ($container->hasDefinition('security.access.decision_manager') && $container->hasDefinition('debug.security.access.decision_manager')) {
```

Comment:
i did this first, but it didn't work.

now i did some debugging, and i guess this didn't work because, this is a decorator service.

in this case, this services takes precedence in order:

- `security.access.decision_manager`
- `debug.security.access.decision_manager`
- `debug.security.access.decision_manager.inner` (which is alias for `security.access.decision_manager`)